File: 215-kth-largest-element-in-an-array/kth-largest-element-in-an-array.java
class Solution {
    public int findKthLargest(int[] nums, int k) {
        PriorityQueue<Integer> pq=new PriorityQueue<>(Collections.reverseOrder());
        for(var i:nums)pq.add(i);
        for(int i=1;i<k;i++)pq.poll();
        return pq.poll();
    }
}

File: 215-kth-largest-element-in-an-array/kth-largest-element-in-an-array.py
class Solution:
    def findKthLargest(self, nums: List[int], k: int) -> int:
        h=[]
        # heappush
        for i in nums:
            heapq.heappush(h,i)
        k=len(nums)-k
        while k>0:
            k-=1
            heapq.heappop(h)
        return heapq.heappop(h)